package.json

```diff
@@ -64,6 +64,7 @@
     "lodash.merge": "^4.6.2",
     "lodash.pick": "^4.4.0",
     "lodash.snakecase": "^4.1.1",
+    "nostr-tools": "^2.1.0",
```

Comment:
Yarn.lock was not committed, needs to be reviewed (There is a wasm package introduced by nostr-tools)